### PR TITLE
& $item Codestyle correction

### PR DESCRIPTION
**Pull request by N6REJ**
Pull Request for Issue # .

### Summary of Changes

Code Style requires no spaces between & $item 

### Testing Instructions

code review

### CODE REVIEW

**Comment by brianteeman** [comment]
@N6REJ the .gitignore file should not be part of the pr

**Comment by N6REJ** [comment]
I THINK I fixed it, I noticed that it added it after I created the  PR , sorry.

**Comment by brianteeman** [comment]
no you managed to delete the file completely - oops

**Comment by N6REJ** [comment]
ok, so, how do I fix it?

**Comment by N6REJ** [comment]
just fyi, I just created another branch {[codestyle2](https://github.com/N6REJ/joomla-cms/tree/codestyle2)} and redid the fix and on commit it only committed the 3 changed files but then when I do a pr it looks like its trying to delete/add .gitignore again :(

**Comment by C-Lodder** [comment]
Submitted a PR to your branch: https://github.com/N6REJ/joomla-cms/pull/3

**Comment by N6REJ** [comment]
@C-Lodder ty!  Done 👯‍♂️

File: .gitignore [hunk 1]
```diff
@@ -184,4 +184,3 @@ composer.phar
 
 # Build related
 RoboFile.ini
-
```
Comment (dgrammatiko) [inline .gitignore]: @N6REJ please add an empty line at the end

Comment (C-Lodder) [reply .gitignore]: https://github.com/N6REJ/joomla-cms/pull/4

**Comment by N6REJ** [comment]
@dgt41 @C-Lodder ty guys.

**Comment by zero-24** [comment]
merging thanks.

**Comment by N6REJ** [comment]
do I delete my branch now or wait?

**Comment by C-Lodder** [comment]
@N6REJ yes, you can delete it

**Comment by brianteeman** [comment]
@N6REJ congratulations on the pr. Sorry I went quiet I was speaking at a user grohp

**Comment by N6REJ** [comment]
@C-Lodder how the heck did you fix the dang .gitignore issue?  It's doing it again on another pr I want to make *shoots self with shotgun* [spelling](https://github.com/N6REJ/joomla-cms/tree/spelling)
I even went as far as to delete the repo, local and refork! / reclone
When I go to do the committ it does show its trying to put .gitignore there ( which is stupid ) but it is lined out cause I've unchecked it..
image

**Comment by wojsmol** [comment]
@N6REJ The problem with .gitignore is that netbeans adds to it `/nbproject/private/` - see https://docs.joomla.org/NetBeans_overview `Creating a project and debugging` point 3.3 for a solution.

**Comment by C-Lodder** [comment]
Because you probably created a fork of a branch where you had already removed the file.

When PRs get merged, you need to make sure you update your repo. You can use command line or the Github UI. Or if you raraly contribue PRs, delete your fork and re-fork it completely.

**Pull request merged**